### ISSUE TITLE
2.6 RC1 - Nginx not working after running the script

### ISSUE THREAD

**Issue by JuanCasad0** [bigbluebutton/bbb-install#594]
Hi there!

When trying to update 2.6 to the new RC1, I noticed nginx didn't work. To avoid any possible mistakes, I reinstalled Ubuntu 20.04 from scratch and ran the script from the blank new OS. The result was the same.

This is what I get with  systemctl status nginx:

 nginx.service - A high performance web server and a reverse proxy server
     Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Fri 2023-01-27 23:46:40 CET; 5min ago
       Docs: man:nginx(8)

Jan 27 23:46:40 ****.contaboserver.net systemd[1]: Starting A high performance web server and a reverse proxy server...
Jan 27 23:46:40 ****.contaboserver.net nginx[36647]: nginx: [emerg] bind() to [::1]:82 failed (99: Cannot assign requested address)
Jan 27 23:46:40 ****.contaboserver.net nginx[36647]: nginx: configuration file /etc/nginx/nginx.conf test failed
Jan 27 23:46:40 ****.contaboserver.net systemd[1]: nginx.service: Control process exited, code=exited, status=1/FAILURE
Jan 27 23:46:40 ****.contaboserver.net systemd[1]: nginx.service: Failed with result 'exit-code'.
Jan 27 23:46:40 ****.contaboserver.net systemd[1]: Failed to start A high performance web server and a reverse proxy server.
Jan 27 23:48:11 ****.contaboserver.net systemd[1]: nginx.service: Unit cannot be reloaded because it is inactive.

Also, after finishing the installation, this messages appear:

# Potential problems described below
# Not running:  Nginx
curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (52) Empty reply from server
.curl: (16) OpenSSL SSL_write: Broken pipe, errno 32
... (and goes on)
# Error: Could not connect to the configured hostname/IP address

The result for bbb-conf --status is:

bbb-conf --status
nginx —————————————————► [✘ - failed]
freeswitch ————————————► [✔ - active]
redis-server ——————————► [✔ - active]
bbb-apps-akka —————————► [✔ - active]
bbb-fsesl-akka ————————► [✔ - active]
mongod ————————————————► [✔ - active]
bbb-html5 —————————————► [✔ - active]
bbb-webrtc-sfu ————————► [✔ - active]
kurento-media-server ——► [✔ - active]
bbb-html5-backend@1 ———► [✔ - active]
bbb-html5-backend@2 ———► [✔ - active]
bbb-html5-frontend@1 ——► [✔ - active]
bbb-html5-frontend@2 ——► [✔ - active]
etherpad ——————————————► [✔ - active]
bbb-web ———————————————► [✔ - active]
bbb-pads ——————————————► [✔ - active]
bbb-export-annotations ► [✔ - active]
bbb-rap-caption-inbox —► [✔ - active]
bbb-rap-resque-worker —► [✔ - active]
bbb-rap-starter ———————► [✔ - active]

Any idea would be very much appreciated! Thanks!

**Comment by ffdixon** [comment]
Can you post the output for `nginx -t`.

**Comment by JuanCasad0** [comment]
Hi Fred!

Sure! Here you are:

nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: [emerg] bind() to [::1]:82 failed (99: Cannot assign requested address)
nginx: configuration file /etc/nginx/nginx.conf test failed

**Comment by ffdixon** [comment]
A few questions -- does your server support IPV6?  Is there some other process already bound to local IPV6 address on port 82.

**Comment by JuanCasad0** [comment]
Dear Fred,

That was it!. Before running the script, I had checked everything was OK. As IPv6 was not shown after ip addr | grep inet6,  I managed to add it by editing netplan.yml (I knew the server had an IPv6 available). However, I took for granted that it would be permanent. It was not.... 
Looking deeper, I found that in /etc/sysctl.conf by default it was disabled by default in my server (net.ipv6.conf.all.disable_ipv6=1). So after commenting that line, everything is working fine now. Just took a reboot and running the script again to make it work.

So, if you consider, I think it'd be interesting to add in the documentation a couple of lines to check that "net.ipv6.conf.all.disable_ipv6=1" is not active in /etc/sysctl.conf when verifying the IPv6 status, asking to add a simple # at the beginning of the line to avoid this problem in case the server has an IPv6. For sure, many users will have this problem in the near future, I'm afraid.

Thank you very much!
